frontend/src/scenes/notebooks/Notebook/Notebook.scss

```
@@ -59,6 +59,25 @@
             border-radius: var(--radius);
             padding: 0.2rem;
         }
+
+        .node-phBacklink {
+            padding: 2px 4px;
```

_typically_ we try and use `rem` for these things as you may see elsewhere in the CSS.
```suggestion
            padding: 0.125rem 0.25rem;
```